server/src/test/python/cromwell-az-e2e-test/cromwell_az_e2e_test/az_e2e.py

```diff
@@ -0,0 +1,173 @@
+import requests
+import os
+import json
+import random
+import string
+import uuid
+import time
+
```

Comment:
We talked about this offline but noting here - it would be great if we could host the Python parts of this in https://github.com/broadinstitute/dsp-reusable-workflows and define the GHA here. Then our test, the CBAS test, and the WDS test can more easily share logic and work in similar ways.